mm2src/lp_swap/maker_swap.rs

```diff
@@ -1164,20 +1219,31 @@ impl MakerSwap {
                         self.r().data.maker_payment_lock + 3700
                     );
                 }
-                let transaction = try_s!(
-                    self.maker_coin
-                        .send_maker_refunds_payment(
-                            &maker_payment,
-                            maker_payment_lock,
-                            other_maker_coin_htlc_pub.as_slice(),
-                            &secret_hash.0,
-                            maker_coin_htlc_keypair.private().secret.as_slice(),
-                            &maker_coin_swap_contract_address,
-                        )
-                        .compat()
-                        .await
+                let fut = self.maker_coin.send_maker_refunds_payment(
```

Comment:
Despite the initial requirement and `send_taker_fee` comment, it's ok to `broadcast_p2p_tx_msg` in `recover_funds`. Please keep it here.